src/plugins/auto/README.md

```diff
@@ -0,0 +1,55 @@
+# OpenVINO™ AUTO Plugin
+
+The main responsibility of the AUTO plugin is to provide a unified device that enables developers to code deep learning applications once and deploy them anywhere. 
+
+Other capabilities of the AUTO plugin include:
+* Static device selection, which intelligently loads a network to one device or multiple devices.
+* CPU acceleration to start inferencing while the target device is still loading the network.
+* Model priority support for loading multiple networks to multiple devices.
+
+The component is written in `C++`. If you want to contribute to the AUTO plugin, follow [the common coding style rules](../../../docs/dev/coding_style.md).
+
+## Key contacts
+
+In case of any questions, review and merge requests, contact the [AUTO Plugin maintainer group](https://github.com/orgs/openvinotoolkit/teams/openvino-ie-auto-multi-maintainers).
+
+## Components
+
+The AUTO plugin follows the OpenVINO™ plugin architecture and consists of several main components:
+* Static device selection
+* Infer request pipeline
+* Inference to expose exection device and properties of execution device.
```

Comment:
This section is about physical components: public API, sources, tests and etc.
Take a look for example on core component:
https://github.com/openvinotoolkit/openvino/tree/master/src/core

Reply:
Updated. Since AUTO follow standard plugin architecture, the common components of standard plugin are out of scope.